js/repl/Repl.js

```diff
@@ -504,6 +507,54 @@ class Repl extends React.Component<Props, State> {
       });
   };
 
+  _onDownloadRepl = (e: SyntheticEvent<*>) => {
+    const {
+      externalPlugins,
+      sourceType,
+      envConfig,
+      presets,
+      runtimePolyfillState,
+    } = this.state;
+    const zip = new JSZip();
+    const folder = zip.folder("my-babel-repl");
+
+    const packageJson = {
+      name: "my-babel-repl",
+      private: true,
+      version: "1.0.0",
+      main: "lib/index.js",
+      scripts: {
+        babel: "babel src --out-dir lib",
+      },
+      devDependencies: externalPlugins.reduce((prev: Object, plugin) => {
+        prev[plugin.name] = plugin.version;
+        return prev;
+      }, {}),
+      babel: baseBabelConfig({
+        sourceType,
+        envConfig,
+        presets,
+        plugins: externalPlugins.map(plugin => plugin.name),
+        sourceMap: runtimePolyfillState.isEnabled,
+      }),
+    };
+
+    packageJson.devDependencies["@babel/core"] = "latest";
```

Comment:
It is using the latest core version but should use the last one available for the selected minor version. Still needs some logic for this.